app/components/lazy-minted/LazyMinted.tsx

```
@@ -0,0 +1,100 @@
+                key={i + '-' + nft.id}
+                onClick={() => handleViewMore(nft)}
+                className="hover:cursor-pointer hover:bg-slate-100 dark:hover:bg-slate-900"
+              >
+                <td className="border border-slate-700 px-4 py-1">
+                  {nft.id.toLocaleString()}
+                </td>
+                <td className="border border-slate-700 px-4 py-1">
+                  <video
+                    src={parseIpfsUri(nft.metadata.animation_url!!)}
+                    width={180}
+                  ></video>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)